src/Matcher.js

```diff
@@ -125,10 +125,18 @@ export default class Matcher {
       params[name] = value && decodeURIComponent(value);
     });
 
-    return {
-      params,
-      remaining: pathname.slice(match[0].length),
-    };
+    let remaining = pathname.slice(match[0].length);
+    if (this.endsWithDelimiter(match[0])) {
```

Comment:
Well it'd make the syntax for the nav very slightly easier in the examples above.